### DISCUSSION TITLE
plugin for Description Lists

### DISCUSSION THREAD

**Discussion by atwak** [remarkjs/remark#848] [Ideas] [answered]
Some processors apply it with this syntax

```
term
: description
```

… with different implementations, allowing either several `<dt>`’s, or `<dd>`’s, like so

```
term 1
term 2
: description 1
: description 2
```

Usually different sets of terms are divided by empty line

```
term 1
: description

term 2
: description
```

And second term in such example wouldn’t be parsed correctly without empty line. Maybe it can be.

**Comment by ChristianMurphy** [comment, marked as answer]
This can be supported with syntax plugins, such as:

https://github.com/Symbitic/remark-plugins/blob/master/packages/remark-deflist/README.md
https://github.com/wataru-chocola/remark-definition-list
